results/benchmark1b.1.md

```diff
@@ -0,0 +1,149 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.2-dev
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```python papermill={"duration": 0.026347, "end_time": "2023-03-07T17:19:41.523290", "exception": false, "start_time": "2023-03-07T17:19:41.496943", "status": "completed"} tags=["parameters"]
```

Comment:
@tkphd, this papermill metadata changes everytime the notebooks are rebuilt, which is very  annoying. Unfortunately, I can't switch off Jupytext saving the metadata as we need the parameters tag for the parameter injection. Very annoying.

Reply:
Jupytext has thankfully removed all of the Jupyter metadata with times and unique IDs that keeps changing. Unfortunately, papermill is doing the same thing.

Reply:
What if we load [`papermill-jupytext`](https://pypi.org/project/papermill-jupytext/) instead of vanilla `papermill`?

Reply:
`papermill-jupytext` doesn't inspire me as it has [18 commits and hasn't been touched for four years](https://github.com/mwouts/papermill_jupytext).